src/edu/washington/escience/myria/api/encoding/QueryConstruct.java

```diff
@@ -452,41 +455,47 @@ private static void assignWorkersToFragments(final List<PlanFragmentEncoding> fr
   }
 
   /**
+   * Instantiate operators in the given fragment.
+   * 
+   * @param planFragment the encoded plan fragment.
+   * @param args args
+   * @param allOperators a map to keep instantiated operators.
+   */
+  private static void instantiateOperators(final PlanFragmentEncoding planFragment, final ConstructArgs args,
```

Comment:
Might be clearer to call this `instantiateOperatorsFromFragment` or `instantiateOperatorsForFragment` or `instantiateFragmentOperators`?